File: android/apolloui/src/main/java/io/muun/apollo/presentation/ui/recovery_code/show/ShowRecoveryCodeView.java
package io.muun.apollo.presentation.ui.recovery_code.show;


import io.muun.apollo.domain.libwallet.RecoveryCodeV2;
import io.muun.apollo.presentation.ui.base.SingleFragmentView;

public interface ShowRecoveryCodeView extends SingleFragmentView {

    void setRecoveryCode(RecoveryCodeV2 recoveryCode);
}

File: android/apolloui/src/main/java/io/muun/apollo/presentation/ui/recovery_code/show/ShowRecoveryCodeView.kt
package io.muun.apollo.presentation.ui.recovery_code.show

import io.muun.apollo.domain.libwallet.RecoveryCodeV2
import io.muun.apollo.presentation.ui.base.SingleFragmentView

interface ShowRecoveryCodeView : SingleFragmentView {

    fun setRecoveryCode(recoveryCode: RecoveryCodeV2)
}